debian/changelog

```diff
@@ -1,3 +1,10 @@
+python-ev3dev (1.1.0) testing; urgency=medium
```

Comment:
@WasabiFan, as I understood from https://github.com/rhempel/ev3dev-lang-python/issues/381#issuecomment-332241734, `testing` here is enough for the package to end up in the stretch repo.

Reply:
Wrong link. Should be https://github.com/rhempel/ev3dev-lang-python/issues/339#issuecomment-327022503

Reply:
I was thinking of `debian/compat`, but it looks like that's already pointing to Stretch..m